docs/user-guide/horizontal-pod-autoscaler.md

```diff
@@ -33,14 +32,20 @@ Further details of the autoscaling algorithm are given [here](https://github.com
 Autoscaler uses heapster to collect CPU utilization.
 Therefore, it is required to deploy heapster monitoring in your cluster for autoscaling to work.
 
-Autoscaler accesses corresponding replication controller or deployment by scale sub-resource.
+Autoscaler accesses corresponding replication controller, deployment or replica set by scale sub-resource.
 Scale is an interface which allows to dynamically set the number of replicas and to learn the current state of them.
 More details on scale sub-resource can be found [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/horizontal-pod-autoscaler.md#scale-subresource).
```

Comment:
I've just noticed this is contains part of the template. How can I test if this will generate correct link?